webapp/services/update_user.py

```diff
@@ -0,0 +1,33 @@
+import webapp2
+import json
+import logging
+from google.appengine.ext import ndb
+from google.appengine.api import search
+from models.user import User
+
+class UpdateUser(webapp2.RequestHandler):
+
+    def post(self):
+
+		json_string = self.request.body
+		dict_object = json.loads(json_string)
+
+		user_id = int( dict_object['userId'] ) 
+		user_name = dict_object['userName']
+		user_email = dict_object['userEmail']
+		user_date_of_birth = dict_object['userDateOfBirth']
+
+		user = User.get_by_id(user_id)
+
+		if user != None:
+			user.name = user_name
+			user.email = user_email
+			user.date_of_birth = user_date_of_birth
+
+			user.put()
+
+			res = { "msg" : "User successfully updated", "success": True, "user_id" : user_id }
+			self.response.out.write(json.dumps(res))
```

Comment:
and here

Reply:
Using all of these responses as output messages in the layout for error / success alerts. Is there a reason why you want me to update all of this? I am aware of REST protocol but even for ajax a post response in json format is super handy.